### ISSUE TITLE
Callback can be called more than once

### ISSUE THREAD

**Issue by PsiXdev** [fleg/mongo-each#2]
I have some examples, when callback called more than once:

**First is:**
```
each(collection.find(), function(doc, callback) {
	callback(new Error());
 function(err) {
	console.log('It log will be called 2 times');
	done();
});
```

**Second is:**
```
each(collection.find(), function(doc, callback) {
	setTimeout(function() {
		callback(new Error());
	}, 1);
}, function(err) {
	console.log('It log will be called many times'); // collection.count() times?
	done();
});
```

**Comment by fleg** [comment]
fixed in https://github.com/fleg/mongo-each/commit/e7a7000f0929770a852abf15a5de99833674d021

**Comment by fleg** [comment]
published as 1.1.2
